benches/filters.rs

```diff
@@ -13,15 +13,21 @@ fn filters_16_bits_filter_0(b: &mut Bencher) {
     let input = test::black_box(PathBuf::from("tests/files/rgb_16_should_be_rgb_16.png"));
     let png = PngData::new(&input, &Options::default()).unwrap();
 
-    b.iter(|| png.raw.filter_image(RowFilter::None, false));
+    b.iter(|| {
+        png.raw
+            .filter_image(FilterStrategy::Basic(RowFilter::None), false)
```

Comment:
Using the basic filters is more verbose now. There are a couple of options for improving this if we feel the need:
1. Implement `From<RowFilter>` on `FilterStrategy` so we can write:
`RowFilter::None.into()`
3. Add constants on `FilterStrategy` like `NONE = Self::Basic(RowFilter::None)`, so we can write:
`FilterStrategy::NONE`

Reply:
I've implemented the second option for now, but open to suggestions.